test/check-loopback

```diff
@@ -42,7 +54,8 @@ class TestLoopback(MachineCase):
         b.wait_visible('#login-fatal')
         self.assertIn("The server refused to authenticate 'admin' using password authentication, and no other supported authentication methods are available.", b.text('#login-fatal'));
 
-        self.allow_journal_messages("cockpit-polkit: couldn't respond to polkit daemon: GDBus\\.Error:org\\.freedesktop\\.PolicyKit1\\.Error\\.Failed: No session for cookie")
+        self.allow_journal_messages("cockpit-polkit: couldn't respond to polkit daemon: GDBus\\.Error:org\\.freedesktop\\.PolicyKit1\\.Error\\.Failed: No session for cookie",
+                                    ".*server offered unsupported authentication methods: public-key.*")
```

Comment:
We should figure out a bit more what is going on here.

The message really means "server offered _only_ unsupported methods", which isn't true for Debian: it has offered the password method, which we support, but there we didn't try because cockpit-ws didn't have a password to use.

This code is inconsistent re password and public-key there: public-key is always considered supported (when compiled in), even if there is no key in cockpit-ws (or the agent) that could be used.  On Fedora, where we do support public-key, we thus never see this message.

I'll propose this patch:

```
diff --git a/src/ws/cockpitsshtransport.c b/src/ws/cockpitsshtransport.c
index e0371c5..96e256b 100644
--- a/src/ws/cockpitsshtransport.c
+++ b/src/ws/cockpitsshtransport.c
@@ -578,6 +578,7 @@ cockpit_ssh_authenticate (CockpitSshData *data)
         }
       else if (!has_creds)
         {
+          methods_tried = methods_tried | method;
           result_string = "not-provided";
         }
       else
```

@petervo, what do you think?